modules/installation-osp-dpdk-selecting-flavors.adoc

```diff
@@ -0,0 +1,45 @@
+:_content-type: PROCEDURE
+[id="installation-osp-dpdk-selecting-flavors_{context}"]
+= Selecting the huge pages size and CPU policy for virtual machines on OVS-DPDK hosts
+
+Each virtual machine that runs on a host that uses Open vSwitch with the Data Plane Development Kit (OVS-DPDK) requires huge pages and a dedicated CPU policy. Before you install a cluster that uses OVS-DPDK, create an {rh-openstack-first} flavor that meets this requirement.
+
+.Prerequisites
+
+* Check with your {rh-openstack} administrator to determine the maximum size and number of huge pages to use for each node.
+
+.Procedure
+
+* Define a flavor that has appropriate specifications: 
+** If you have administrative access to the {rh-openstack} deployment:app-name: 
+... From a command line, create a flavor: 
++
+[source,terminal]
+----
+$ openstack flavor create m1.xlarge.nfv --ram 16384 --disk 40 --vcpus 8
+----
+... Set a dedicated CPU policy for that flavor:
++
+[source,terminal]
+----
+$ openstack flavor set --property hw:cpu_policy=dedicated --property hw:mem_page_size=large m1.xlarge.nfv
+----
+
+** If you do not have administrative access to the {rh-openstack} deployment, define flavor properties in the `install-config.yaml` file for your cluster in the `platform.openstack.clusterOSImageProperties` object. For example:
++
+[source,yaml]
+----
+...
+    platform:
+      openstack:
+         clusterOSImageProperties:
+            - hw_cpu_policy: dedicated
+            - hw_mem_page_size: large
+----
+
+[NOTE]
+====
+If you use the image properties method, the flavor that you use to deploy virtual machines must allow the image property values that you select. 
+
+For example, you cannot deploy a VM that uses an image with the property `hw:mem_page_size: 1GB` with a flavor that has a `hw:mem_page_size` value of `small` because the `small` size is incompatible with a 1GB page size.
```

Comment:
- Here, you use "VM" but elsewhere in the file, you spell out "virtual machine". I think I've seen us just cut to "VM" without defining it but your call.
- s/1GB/1 GB

Reply:
```suggestion
For example, you cannot deploy a VM that uses an image with the property `hw:mem_page_size: 1GB` with a flavor that has a `hw:mem_page_size` value of `small` because the `small` size is incompatible with a 1 GB page size.
```